internal/install/recipe_installer.go

```diff
@@ -100,11 +101,20 @@ func (i *recipeInstaller) install() error {
 		}
 	} else {
 		// Ask the recipe service for recommendations.
-		var err error
+		log.Debugln("Fetching recommendations.")
```

Comment:
I think this line isn't needed.  The `fetchRecommendataions` might be spinning on this already.

Reply:
👍